spi/identity-hub-spi/src/main/java/org/eclipse/dataspaceconnector/identityhub/credentials/VerifiableCredentialsJwtService.java

```diff
@@ -45,4 +50,26 @@ public interface VerifiableCredentialsJwtService {
      * @return VerifiableCredential represented as {@code Map.Entry<String, Object>}.
      */
     Result<Map.Entry<String, Object>> extractCredential(SignedJWT jwt);
+
+    /**
+     * Builds a verifiable credential as a signed JWT
+     *
+     * @param credential The verifiable credential to sign
+     * @param issuer The issuer of the verifiable credential
+     * @param subject The subject of the verifiable credential
+     * @param jwk The private key of the issuer, used for signing
+     *
+     * @return The Verifiable Credential as a JWT
+     * @throws Exception In case the credential can not be signed
+     */
+    SignedJWT buildSignedJwt(VerifiableCredential credential, String issuer, String subject, ECKey jwk) throws Exception;
+
+    /**
+     * Reads an EC key from a PEM file
+     *
+     * @param file The PEM file
+     * @return The EC Key
+     * @throws Exception In case the key can not be read
+     */
+    ECKey readEcKey(File file) throws Exception;
```

Comment:
It seems odd to have a service method here for this.
If this method is really justified, it should take a `String` argument (or similar) so that it's the caller that deals with file I/O problems.
Otherwise, based on the feedback above, the method can probably be deleted and the CLI caller should instantiate an `EcPrivateKeyWrapper`

Reply:
I moved the method to a utility class as its used in tests as well